rfcfold

```diff
@@ -282,7 +282,7 @@ unfold_it() {
   fi
   dbg "nothing to do"
   cp "$infile" "$outfile"
-  return -1
+  return 255
```

Comment:
Curious, why is -1 not okay?   Is it defined to be an unsigned value?

Assuming it should be changed, do you think 255 is better than, say, '2'?

Reply:
The exit code is an unsigned integer less than 256 (legal values are from 0 to 255). If a process is terminated by an unhandled signal, the signal number affects the exit code. On Linux, the exit code is _128 + signal number_, thus I would prefer to use exit code values below 128.

Writing `-1` to express `255` works in Bash, but not in, e.g., Dash. As such, I do not change the value, only its representation. I think it is confusing to write `-1` when the value can only be read as `255`.

I'd prefer `2` to `255` if you want to change the actual value.

Reply:
Yes, I think `2` is better than `255`.

I wonder also about replacing:
```
0: success (folded)
1: error
2: no-op (not-folded)
```

With:
```
0: success (folded)
1: no-op (not-folded)
2: error
```

But I don't see any upside to it.   We might view "no-op" as a variant of `success`, but I thought it might be helpful for scripts, rather than grep-ing the first line to determine that, if it is ever important at all...

Reply:
I would use `2` rather than `1`  for the special "_error_" of not folding because `1` is commonly used for a general error, different non-zero values for specific errors.

I concur that using a specific exit code for _no-op_ is helpful, as can be seen in the tests.